src/uct/ib/base/ib_md.c

```diff
@@ -129,6 +129,11 @@ static ucs_config_field_t uct_ib_md_config_table[] = {
    "Prefer nearest device to cpu when selecting a device from NET_DEVICES list.\n",
    ucs_offsetof(uct_ib_md_config_t, ext.prefer_nearest_device), UCS_CONFIG_TYPE_BOOL},
 
+  {"CONTIG_PAGES", "n",
+   "Enable allocation with contiguous pages. Warning: enabling this option may\n"
+   "cause stack smashing.\n",
```

Comment:
warning is too destructive

why not to use contig allocation by default ON for fixed addr only (and take care for VA with mmap and set driver contig usage for ANON) ?